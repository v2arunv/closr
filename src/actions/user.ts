import {IUser} from "../models/users";

export interface IUserAction {
    type: string,
    user: IUser,
}

export const setUser = (user: IUser): IUserAction => {
    return <IUserAction> {
        type: 'SET_USER',
        user: user,
    }
};