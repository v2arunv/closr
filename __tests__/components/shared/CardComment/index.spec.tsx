import React from 'react';
import CardComment from '@sharedComponents/CardComment';
import renderer from 'react-test-renderer';
import user from '@mocks/users';

test('renders correctly', () => {
    const tree = renderer.create(
        <CardComment
            username={'user'}
            commentText={'comment'}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});