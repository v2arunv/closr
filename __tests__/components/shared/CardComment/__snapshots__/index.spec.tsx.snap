// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "padding": 10,
    }
  }
>
  <View
    style={
      Object {
        "height": 30,
        "width": 30,
      }
    }
  >
    <Image
      defaultSource={
        Object {
          "testUri": "../../../src/assets/images/dp_placeholder.jpg",
        }
      }
      source={
        Object {
          "uri": undefined,
        }
      }
      style={
        Object {
          "borderRadius": 12.5,
          "height": 25,
          "padding": 5,
          "width": 25,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#c1d0ff",
        "borderRadius": 10,
        "marginLeft": 5,
        "padding": 5,
        "width": "70%",
      }
    }
  >
    <Text
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      user
    </Text>
    <Text
      style={
        Object {
          "marginTop": 5,
        }
      }
    >
      comment
    </Text>
  </View>
</View>
`;
